lorekeeper.gemspec

```diff
@@ -34,4 +34,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rbtrace', '~> 0.4'
   spec.add_development_dependency 'simplecov', '~> 0.16'
   spec.add_development_dependency 'mutant-rspec', '~> 0.8'
+  spec.add_development_dependency 'rubocop-mdsol', '~> 0.3'
```

Comment:
we love `rubocop-performance`. can you add it here?
```
spec.add_development_dependency "rubocop-performance", "~> 1.14"
```

and add this to the top of `.rubocop.yml`?
```
require:
  - rubocop-performance
```

Reply:
👍  this is performance sensitive code so yeah bring the goods